src/pcr.c

```diff
@@ -203,11 +203,17 @@ void
 pcr_bank_init_from_snapshot(tpm_pcr_bank_t *bank, const char *efivar_path)
 {
 	FILE *fp;
+	char buf[4];
 
 	debug("Trying to find PCR values in %s\n", efivar_path);
 	if (!(fp = fopen(efivar_path, "r")))
 		fatal("Unable to open \"%s\": %m\n", efivar_path);
 
+	/* The efivarfs files are not seekable. Use fread() to skip over
+	 * 4 bytes of variable attributes
+	 */
+	fread(linebuf, 4, 1, fp);
```

Comment:
Shouldn't this be "buf" rather than linebuf?

Reply:
Sorry, I accidentally pushed the intermediate patch. Fixed and force-pushed.